build-system/compile/pre-closure-babel.js

```diff
@@ -60,27 +45,62 @@ function getFilesToTransform() {
  * @return {!Promise}
```

Comment:
Replace "an in-memory cache" with "an on-disk cache" in this JSDoc.

Reply:
Reverted.